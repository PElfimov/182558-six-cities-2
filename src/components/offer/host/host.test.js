import React from 'react';
import renderer from 'react-test-renderer';
import Host from './host';

const host = {
  id: 5,
  name: `Nadia`,
  isPro: true,
  avatarUrl: `/fff.jpg`};
const description = `bla bla `;

it(`Host correctly renders after relaunch`, () => {
  const tree = renderer
    .create(
        <Host
          host={host}
          description = {description}
        />).toJSON();
  expect(tree).toMatchSnapshot();
});
