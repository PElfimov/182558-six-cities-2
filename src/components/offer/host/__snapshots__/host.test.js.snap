// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Host correctly renders after relaunch 1`] = `
<div
  className="property__host"
>
  <h2
    className="property__host-title"
  >
    Meet the host
  </h2>
  <div
    className="property__host-user user"
  >
    <div
      className="property__avatar-wrapper property__avatar-wrapper--pro user__avatar-wrapper"
    >
      <img
        alt="Host avatar"
        className="property__avatar user__avatar"
        height="74"
        src="//fff.jpg"
        width="74"
      />
    </div>
    <span
      className="property__user-name"
    >
      Nadia
    </span>
    <span
      className="property__user-status"
    >
      Pro
    </span>
  </div>
  <div
    className="property__description"
  >
    <p
      className="property__text"
    >
      bla bla 
    </p>
  </div>
</div>
`;
